tracker_visp/location location
geometry_msgs/PoseStamped pose